README.md

```diff
@@ -1,2 +1,21 @@
-# hw01
-Repository for first CFSS homework assignment
+# **Introduction**
+
+## *Name*
+Xian Qu
+
+## *Program*
+MAPSS
+
+## *Personality*
+1. Outgoing
+2. Hardworking
+3. Open-minded:
+   - towards people
+   - towards life
+
+## *Picture*
+![Image of Snoopy](https://s-media-cache-ak0.pinimg.com/originals/00/45/96/004596e8d225dae6edb2f727cc33fe41.png)
+
+##*Questions and Reflections*
+1. The order of emphasis and header: figured out that header code should go first and then emphasis
+2. This assignment is quite interesting. I went back to the installation steps before to help myself remember and then start new steps.Also, besides steps on Github website of Markdown, it could also work from R Markdown. 
```

Comment:
Actually, if  you use R Markdown, you will get a table of information (tilte, author, date...) on top of the page. (just like my hw01)
You can also try insertting a web link (if you don't count the link of the picture)